### DISCUSSION TITLE
https://imagekit.io/docs/api-reference/digital-asset-management-dam/managing-assets/get-file-details

### DISCUSSION THREAD

**Discussion by giscus** [imagekitio/discussions#38] [Docs]
# https://imagekit.io/docs/api-reference/digital-asset-management-dam/managing-assets/get-file-details

This API returns an object with details or attributes about the current version of the file.

https://imagekit.io/docs/api-reference/digital-asset-management-dam/managing-assets/get-file-details

**Comment by fichaepi** [comment]
Since sometime in August 2024 "url" field get this "?updateAt=xxxxxx" appended..... Please restore the original way it used to be (without updatedAt).

UpdatedAt is already present in the response body, so why duplicating this info and messing up with the url?  This is causing us headaches since we used to use "url" to sign the image url.  It apparently doesnt accepting the url when "?updatedAt=xxxx" is there , so now we have to rework ALL urls excluding this nasty ?updatedAt=xxxxxxx ... else allow a new request parameter to make it optional , for those who may eventually need it like this.

**Reply by ahnv** [reply]
Hey @fichaepi,

We would like to understand why this change is causing an issue at your end, could you elaborate how updatedAt is causing issues in signature signing?

Our SDK's provide you with a simple function to generate signed url with the urls returned by the API, for example,
```
var imageURL = imagekit.url({
    src : "https://ik.imagekit.io/your_imagekit_id/endpoint/default-image.jpg?updatedAt=1726045259772",
    signed: true
});
```

Also, could you help us with your ImageKit ID?

**Comment by avila17tony-oss** [comment]
antoniomarquezavila2@gmail
Com
